### PR TITLE
Bump classnames from 2.2.5 to 2.3.2

### PR DESCRIPTION
**Pull request by levadadenys**
### Bump classnames from 2.2.5 to 2.3.2
Simple bump of `classnames` lib version.
No breaking changes intended, just a small improvement for Typescript dev experience. (In this version lib has types for TS, previously it didn't have them)
Now we can use `import classNames from "classnames";` instead of `const classNames = require('classnames');`
## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]
🎉

File: apps/src/music/views/ChordPanel.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import React, {useState, useEffect, useCallback} from 'react';
+import classNames from "classnames";
```
Comment (breville) [inline apps/src/music/views/ChordPanel.tsx]: Looks like we use single quotes.

**Review by breville — APPROVED** [review]
Not very familiar with this component, but Music Lab changes look fine.  Thanks!

**Review by sanchitmalhotra126 — APPROVED** [review]
Thanks for doing this!

**Pull request merged**